inventory/views.py

```diff
@@ -49,8 +48,8 @@ def make_station_order(request, order_pk):
     station_order = get_object_or_404(StationOrder, pk=order_pk)
     station = station_order.station
     station_name = station.get_name()
-    station_id = station.station_id
-    station_fleet = Vehicle.objects.filter(station=station_id)
+    station_pk = station.pk
```

Comment:
Do you believe that `station_pk` is more descriptive, than `station.pk`?

Reply:
If I need to use the value of `station.pk` multiple times within the function - I'd rather assign a variable to it than calling the `station` object.